BrokenKeyboard.scala

```diff
@@ -0,0 +1,32 @@
+import BrokenKeyboard.WordStorage
+
+object BrokenKeyboard extends App {
+  import scala.io.Source
+  var Characters = Array("a", "b", "c", "d")
```

Comment:
variables start with a lowercase letter and only subsequent words have first letter capitalised, you have quite a few variables that are like this